src/components/upgrader/boxfishUpgraderFactory.ts

```diff
@@ -5,10 +5,14 @@ import IDeviceUpgrader from './../../interfaces/IDeviceUpgrader';
 import BoxfishUpgrader from './../upgrader/boxfishUpgrader';
 import HPKUpgrader from './../upgrader/hpkUpgrader';
 
+import BoxfishPkg from './boxfishpkg';
+import BoxfishHpk from './boxfishhpk';
+
 import { EventEmitter } from 'events';
 
-const HPK_SUPPORT_VERSION = '1.1.0-0';
-export default async function createBoxfishUpgrader(
+export const HPK_SUPPORT_VERSION = '1.1.0-0';
```

Comment:
We should probably bump kubicam-sw version to 1.2.0. The interface has been extended (camera/get_state).